README.md

```diff
@@ -207,9 +208,9 @@ The graphql playground can now be accessed at [localhost:4001](http://localhost:
 It may take a long time to build dependencies when launching servers first time.
 
 ```sh
-$ docker build --target dev-ui -t photoview-ui .
-$ cp ./ui/example.env ./ui/.env
-$ docker run --rm -it -v `pwd`:/app --network host photoview-ui # Monitor source code and (re)launch UI server
+$ docker build --target ui -t photoview/ui . # Build image for development
+$ docker run --rm -it -v `pwd`:/app --network host --env-file ui/example.env photoview/ui \
```

Comment:
Same question regarding the `--network host` here. I see that it was introduced by some of your previous merges, but I probably didn't notice this earlier...

Reply:
It requires to be in the same network with API server.

Reply:
I'll resolve this thread, as we discuss more-less the same in the other one